manifests/stub.pp

```diff
@@ -4,9 +4,14 @@
 #
 define unbound::stub (
   $address,
+  $hostname,
```

Comment:
Leaving no default values will force the user to set both of them, causing the failure below every time.